docs/advanced-use/migrating-from-self-hosted-to-cloud.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Migrating from Self Hosted to Cloud
+---
+
+Flagsmith offers some helpful utilities if you want to migrate your application from your self hosted instance to our
+cloud offering.
+
+The process looks something like:
+
+Step 1. Contact Flagsmith support to confirm you would like to migrate from self hosted to cloud Step 2. Generate a JSON
```

Comment:
Maybe we should break every step into its own line?

Reply:
Whoops, yeah, looks like prettier didn't like me having things on new lines.